### PR TITLE
Fix for aeson-2

### LINKED ISSUE #258: Build error with aeson-2
When I build `shakespeare-2.0.25` with `aeson-2.0.1.0`, I get the following error:

```
Text/Julius.hs:120:23: error:
    • Couldn't match type ‘Data.Aeson.KeyMap.KeyMap Value’
                     with ‘H.HashMap T.Text Value’
      Expected type: H.HashMap T.Text Value
        Actual type: aeson-2.0.1.0:Data.Aeson.Types.Internal.Object
    • In the first argument of ‘H.toList’, namely ‘m’
      In the expression: H.toList m
      In the expression:
        case H.toList m of
          (x : xs) -> singleton '{' <> one x <> foldr f (singleton '}') xs
          _ -> "{}"
    |
120 |         case H.toList m of
    |                       ^
```

### PR DESCRIPTION
**Pull request by brandon-leapyear**
Resolves #258 

Not sure how to / if I should add aeson-2 to the CI matrix, but it builds locally

### CODE REVIEW

**Review by snoyberg — APPROVED** [review]
Thanks!

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -15,6 +15,10 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
         resolver: [nightly, lts-18, lts-16, lts-14]
+        includes:
+          - os: ubuntu-latest
+            resolver: nightly
+            extra_args: --stack-yaml=stack-ghc-9.2.yaml
```
Comment (snoyberg) [inline .github/workflows/tests.yml:21]: Actually, on closer inspection, it looks like the builds are reporting using this for _all_ builds now. I'd recommend unifying this with `resolver` as something like:

```yaml
resolver:
- --resolver nightly
- --resolver lts-18
- ...
- --stack-yaml=stack-ghc-9.2.yaml
```

**Review by snoyberg — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (brandon-leapyear) [reply .github/workflows/tests.yml:21]: Sorry, can you try it again? I used `includes` instead of `include` 🤦

Comment (brandon-leapyear) [reply .github/workflows/tests.yml:21]: Weird... I'm not sure why it's not working. But i'll change it anyway

**Comment by snoyberg** [comment]
Thanks!

**Pull request merged**